### DISCUSSION TITLE
onClick attribute not working on latest release of next( version: 13.4.17)

### DISCUSSION THREAD

**Discussion by MananMaheshwari** [vercel/next.js#54196] [Help]
### Summary

I am not able to use attributes like onClick or onChange or onMouseEnter. I confirmed that when i changed the version to 13.4.13, I am able to use these attributes but not working for next version 13.4.17.
Is this with me only or everyone facing the same issue?

**Comment by shulinnn** [comment]
Hello. I'm also facing issue where no onClick handler is working for me.
```
{
  "name": "wbgl-league-app",
  "version": "0.1.0",
  "private": true,
  "scripts": {
    "dev": "next dev",
    "build": "next build",
    "start": "next start",
    "lint": "next lint"
  },
  "dependencies": {
    "autoprefixer": "10.4.15",
    "eslint": "8.47.0",
    "eslint-config-next": "13.4.19",
    "next": "13.4.19",
    "postcss": "8.4.28",
    "react": "18.2.0",
    "react-dom": "18.2.0",
    "tailwindcss": "3.3.3"
  }
}
```
page.js after npx next@latest project creation

```
"use client";

export default function Home() {
  return (
    <main>
      <button onClick={() => console.log("XD")}>iok</button>
    </main>
  );
}
```

**Reply by icyJoseph** [reply]
It works for me. Is that all of the code you have?

**Reply by shulinnn** [reply]
Its just fresh next app , with nothing else changed , so its irrelevant

**Reply by icyJoseph** [reply]
It is relevant because I did just that and the handler worked. 

Do you see this in incognito mode or any particular browser? What OS have you got?

**Reply by shulinnn** [reply]
Using Brave with Windows 11. Also when i change my next version to 13.4.16 it works as intended.

**Reply by shulinnn** [reply]
@icyJoseph if clone the repo does it work for you ? https://github.com/shulinnn/next-reprod

**Reply by icyJoseph** [reply]
I'll check after my lunch break. Very suspicious issue 🤔

**Reply by shulinnn** [reply]
Thanks.

**Reply by icyJoseph** [reply]
@shulinnn I went here: https://stackblitz.com/github/shulinnn/next-reprod, and it worked - do I have to build the app and start it?

**Reply by icyJoseph** [reply]
Sometimes Stackblitz can't start for some reason, but here's a screen recording, where it works:

https://github.com/vercel/next.js/assets/21013447/b293a35a-5086-46aa-8c07-1f7b93ca75ea

**Comment by icyJoseph** [comment]
Alright yall! The solution is to upgrade to `Node.js` 16.14.0 or later. Preferably 18.16.x or latest (LTS).

The issue is that in 16.13.0, there's an problem with stream/web, yada yada. More here: https://github.com/vercel/next.js/issues/54269

**Comment by T1CkShot** [comment]
Have the same issue on node v18.13.0 and Next v13.4.19

**Reply by T1CkShot** [reply]
Apparently it was an silly mistake on my side. I did not put <html> in layout.
